api/valor_api/backend/core/prediction.py

```diff
@@ -127,12 +127,21 @@ def create_predictions(
         predictions, annotation_ids
     ):
         for i, annotation in enumerate(prediction.annotations):
-            for label in annotation.labels:
+            if annotation.labels:
```

Comment:
is this tested anywhere?

Reply:
What do you want to test here? This code is all covered by tests. There isn’t a functional test or unit test for prediction.py, but we could add one.